ecl/hql/hqlexpr.cpp

```diff
@@ -6614,6 +6615,27 @@ IHqlExpression *CHqlRecord::lookupSymbol(IIdAtom * fieldName)
     return ret;
 }
 
+IHqlExpression * CHqlRecord::lookupNearestSymbol(IIdAtom * fieldName)
```

Comment:
I don't think the versions without the NearestSymbol parameter are called from anywhere, so can be deleted.